Mathlib/Data/Polynomial/UnitTrinomial.lean

```diff
@@ -131,7 +131,7 @@ variable (p q : ℤ[X])
 
 /-- A unit trinomial is a trinomial with unit coefficients. -/
 def IsUnitTrinomial :=
-  ∃ (k m n : ℕ) (hkm : k < m) (hmn : m < n) (u v w : Units ℤ), p = trinomial k m n u v w
+  ∃ (k m n : ℕ) (_hkm : k < m) (_hmn : m < n) (u v w : Units ℤ), p = trinomial k m n (u : ℤ) v w
```

Comment:
```suggestion
  ∃ (k m n : ℕ) (_ : k < m) (_ : m < n) (u v w : Units ℤ), p = trinomial k m n (u : ℤ) v w
```
I prefer not to use `_h` unless actually necessary, but I really don't mind either way.